config/checkstyle_input_suppressions.xml

```diff
@@ -9,6 +9,9 @@
   <!-- Suppress miscellaneous files -->
   <suppress checks="Regexp" files="package-info.java"/>
 
+  <!-- Suppress input files for JavadocMetadataScraper -->
+  <suppress checks="Regexp" files="meta[\\/].*"/>
```

Comment:
please share violations, we should not have violations is such files.

Reply:
This is not needed since its not a check configuration testing
`[ERROR] [checkstyle] [ERROR] /Users/gdasgupt/personal-workspace/checkstyle-dev/checkstyle/src/test/resources/com/puppycrawl/tools/checkstyle/meta/inputs/checks/InputNoCodeInFileCheck.java:0: All check input files in the 'src/(it|test)/resources*/' folder must                    have a comment at the top of the file specifying the check configuration used to                    produce the violations, that begins with "/* Config:". [Regexp]
`

Reply:
thanks a lot!
But problem is deeper.
Please give property `id` to this Check instance in config and please suppress by `id` (not by Check name, suppression by Check name will suppress all violations from all instances)

Reply:
Done